src/Utility/Text.php

```diff
@@ -61,6 +61,9 @@ class Text
     public static function uuid()
     {
         $random = function_exists('random_int') ? 'random_int' : 'mt_rand';
```

Comment:
if random_compat is installed, we can immediately do: 

```php
$random = 'random_int';
```

am i missing something?

Reply:
When upgrading an existing application, there seems to has a short time random_int to not exist, between updating the package with composer.
If we can ignore this, we can unify to random_int.

Reply:
> When upgrading an existing application, there seems to has a short time random_int to not exist, between updating the package with composer.

The composer.json change makes that impossible though, right?

Reply:
I removed the deprecation notice and defaulted to `random_int()`. If PHP suggests this implementation in their official doc, it's good enough for me.

https://www.php.net/manual/en/function.random-int.php